toolkit/scss/forms/_checkbox-tree.scss

```diff
@@ -5,7 +5,7 @@
 
 @include fake-full-width('.checkbox-tree');
 
-fieldset.checkbox-tree__selection-buttons > legend {
+fieldset.checkbox-tree__inputs > legend {
   @extend %visuallyhidden;
```

Comment:
I can see it's not a change in this request, but I'm not sure you should be using extends here instead of a mixin, as it doesn't really make sense as an extension.  I think this should probably go into tech debt though if you are trying to get things out quickly.  I'm sure we were talking about this when I was still on Digital Marketplace.